loops.js

```diff
@@ -33,6 +40,15 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
 
+for (k = 0; k < 16; k++) {
+    if (k === 0) {
```

Comment:
Deze if statement is overbodig
Met de else if controleer je al de even nummers
Met de else statement log je de overige (Oneven)nummers